core/delegate.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Delegate client.
+ *
+ * @module core/delegate
```

Comment:
Not a core. Move somewhere else. It's not a component - it's a nearly-complete client app.